### PR TITLE
Added Database Entities

### PR DESCRIPTION
**Pull request by harshjohar**
Database entities
- users
- events
- organisers
- transactions
- winners
- news

### CODE REVIEW

File: src/main/java/com/pecacm/backend/entities/News.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "organisers", schema = "public")
```
Comment (ken1000minus7) [inline src/main/java/com/pecacm/backend/entities/News.java:11]: This table name seems incorrect

File: src/main/java/com/pecacm/backend/entities/Sponsor.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    private String name;
+
+    @Column(name = "website")
+    private String website;
+
+    @Column(name = "display_logo")
+    private String logo;
+
+    @Column(name = "sponsorship")
+    private String sponsorship;
```
Comment (ken1000minus7) [inline src/main/java/com/pecacm/backend/entities/Sponsor.java:29]: What is this property? 🤔

File: src/main/java/com/pecacm/backend/entities/Organiser.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    @ManyToOne
+    @JoinColumn(name = "event_id")
+    private Event event;
+
+    @Column(name = "work_done")
+    private String work;
```
Comment (ken1000minus7) [inline src/main/java/com/pecacm/backend/entities/Organiser.java:28]: Work done refers to? 🤔

**Review by rawalabhinav — APPROVED** [review]
LGTM

File: src/main/java/com/pecacm/backend/entities/User.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    private String dp;
+
+    @Column(name="student_id")
+    private Integer sid;
+
+    @Column(name = "verified")
+    private Boolean verified;
+
+    @Column(name="designation")
+    private String designation;
```
Comment (ken1000minus7) [inline src/main/java/com/pecacm/backend/entities/User.java:44]: designation what

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]

File: src/main/java/com/pecacm/backend/entities/Sponsor.java (hunk 2 above)
Comment (harshjohar) [reply src/main/java/com/pecacm/backend/entities/Sponsor.java:29]: This property will contain the contract with them, either in form of gdoc or directly what they gave

File: src/main/java/com/pecacm/backend/entities/News.java (hunk 1 above)
Comment (ken1000minus7) [reply src/main/java/com/pecacm/backend/entities/News.java:11]:
```suggestion
@Table(name = "news", schema = "public")
```

File: src/main/java/com/pecacm/backend/entities/Organiser.java (hunk 3 above)
Comment (harshjohar) [reply src/main/java/com/pecacm/backend/entities/Organiser.java:28]: What was done by the given "user" in the given "event", a short description of that. Full description shall be in the event doc.

File: src/main/java/com/pecacm/backend/entities/User.java (hunk 4 above)
Comment (harshjohar) [reply src/main/java/com/pecacm/backend/entities/User.java:44]: Like 
- acm sexy
- ib member
- eb member
- core committee

File: src/main/java/com/pecacm/backend/entities/News.java (hunk 1 above)
Comment (harshjohar) [reply src/main/java/com/pecacm/backend/entities/News.java:11]: Please explain, what exactly seems incorrect @ken1000minus7

**Review by ken1000minus7 — APPROVED** [review]
Nyc work

**Pull request merged**